maestro-cli/src/main/java/maestro/cli/cloud/CloudInteractor.kt

```diff
@@ -153,6 +166,24 @@ class CloudInteractor(
         }
     }
 
+    private fun printDeviceInfo(deviceInfo: DeviceInfo, iOSVersion: String?, androidApiLevel: Int?) {
+
+        val platform = Platform.fromString(deviceInfo.platform)
+
+        val line1 = "Maestro Cloud device specs:\n* ${deviceInfo.displayInfo}"
+        val line2 = "To change OS version use this option: ${if (platform == Platform.IOS) "--ios-version=<version>" else "--android-api-level=<version>"}"
+
+        val version = when(platform) {
+            Platform.ANDROID -> "${androidApiLevel ?: 30}" // todo change with constant from DeviceConfigAndroid
+            Platform.IOS -> "${iOSVersion ?: 15}" // todo change with constant from DeviceConfigIos
+            else -> "n/a"
```

Comment:
What about not even printing the deviceInfo if its not Android/iOS?  Since we do not support anything differently in Maestro Cloud we would avoid printing out things that might confuse someone